Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLMeshObject.cs

```diff
@@ -212,4 +269,81 @@ public void UpdateSkinnedMesh()
             }
         }
     }
+
+    /// <summary>
+    /// RGL counterpart of Unity Texture.
+    /// Contains information about RGL texture.
+    /// </summary>
+    public class RGLTexture
+    {
+        public int Identifier;
+        public Texture2D Texture;
+        public IntPtr rglTexturePtr = IntPtr.Zero;
+
+        public RGLTexture(){}
+        
+        public RGLTexture(Texture2D texture)
+        {
+            Identifier = texture.GetInstanceID();
```

Comment:
I'd move `Identifier` to the constructor parameters. Let high-level scripts decide what is an identifier.